tests/api_resources/test_issuer_fraud_record.py

```diff
@@ -0,0 +1,33 @@
+import stripe
+from tests.helper import StripeTestCase
+
+
+TEST_RESOURCE_ID = 'issfr_123'
+TEST_CHARGE_ID = 'ch_123'
+
+
+class IssuerFraudRecordTest(StripeTestCase):
+    def test_is_listable(self):
+        resources = stripe.IssuerFraudRecord.list()
+        self.assert_requested(
+            'get',
+            '/v1/issuer_fraud_records'
+        )
+        self.assertIsInstance(resources.data, list)
+        self.assertIsInstance(resources.data[0], stripe.IssuerFraudRecord)
+
+    def test_is_retrievable(self):
+        resource = stripe.IssuerFraudRecord.retrieve(TEST_RESOURCE_ID)
+        self.assert_requested(
+            'get',
+            '/v1/issuer_fraud_records/%s' % TEST_RESOURCE_ID
+        )
+        self.assertIsInstance(resource, stripe.IssuerFraudRecord)
+
+    def test_is_retrievable_by_charge(self):
+        resource = stripe.IssuerFraudRecord.retrieve(TEST_CHARGE_ID)
```

Comment:
This probably doesn't take the exact same arguments as retrieving by issuer fraud record ID right?

Reply:
Yup, that's not right. Thanks for catching!